src/styles/content.scss

```
@@ -0,0 +1,99 @@
+$blue: #00acdc;
+$main-accent: #060b35;
+$secondary: #616070;
+
+.main {
+  margin: 0 auto;
+  padding: 50px 0;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: 1000px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+  &__image {
+    width: 160px;
+    margin: 0 auto 40px;
+    height: 134px;
+  }
+  &__name {
+    margin-bottom: 16px;
+  }
```

add new line after each selector

```suggestion
.card {
  display: flex;
  flex-direction: column;
  width: 200px;
  border: 1px solid #f3f3f3;
  padding: 32px 16px 16px;
  border-radius: 5px;
  margin-bottom: 46px;
  
  &:not(:nth-child(4n)) {
    margin-right: 48px;
  }
  
  &__image {
    width: 160px;
    margin: 0 auto 40px;
    height: 134px;
  }
  
  &__name {
    margin-bottom: 16px;
  }
```